server/db/migrate/20150123165155_create_deployments.rb

```diff
@@ -4,9 +4,21 @@ def change
       t.string :name
       t.integer :lifecycle_environment_id
       t.integer :organization_id
-      t.text :rhev_params
-      t.text :cfme_params
-      t.text :openstack_params
+      t.boolean :deploy_rhev
+      t.boolean :deploy_cfme
+      t.boolean :deploy_openstack
+      t.integer :rhev_hypervisor_host_id
+      t.integer :rhev_engine_host_id
```

Comment:
I'm not quite sure how has_one works. To me the Deployment *has* a Host. So would the has_one be on the Deployment? how do I specify the foreign_key?

```
    # TODO: need to figure out the syntax for this
    # has_one :host, foreign_key => :rhev_hypervisor_host_id, dependent => :nullify
    # has_one :host, foreign_key => :rhev_engine_host_id, dependent => :nullify
    #has_one :rhev_hypervisor_host_id
    #has_one :rhev_engine_host_id
```